Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,237 @@
+
+A list of dependencies can be found on [dependency list](TBD). 
+
+# Upgrades and updates
+**TODO**
+
+# Layout on disk
+**TODO**
+
+# Acquisition vectors
+By "vectors" we are refering here to the way any given user can obtain the installers. In the CLI, there are two main planned vectors:
+
+1. [Getting Started Page](https://aka.ms/dotnetcoregs)
+2. [Repo landing page](https://github.com/dotnet/cli/blob/rel/1.0.0/README.md)
+
+There are two more planned vectors:
+
+1. Visual Studio
+2. VS Code 
+
+## Getting Started page
+The page can be found on https://aka.ms/dotnetcoregs.This is the main curated first-run experience for the dotnet CLI. The intent of the page is to help users "kick the tires" quickly and become familiar with what the platform offers. This should be the most stable and curated experience we can offer. The Getting Started page should only point users to curated install experiences that can contain only stable or LKG bits. 
+ 
+
+The below table shows the pertinent information for installs on the "Getting started" page. 
+ 
+| Property              	| Description                                                  	|
+|-----------------------	|--------------------------------------------------------------	|
+| Debian feed           	| Development                                                  	|
+| Local install scripts 	| Latest from rel/1.0.0                                        	|
+
+## Repo landing page
+The repo landing page can be found on: https://github.com/dotnet/cli/readme.md. Download links on the landing page should be decreased in importance. First thing for "you want to get started" section should link to the getting started page on the marketing site. The Repo Landing Page should be used primarily by contributors to the CLI. There should be a separate page that has instructions on how to install both the latest stable as well as latest development with proper warnings around it. The separate page is to really avoid the situation from people accidentally installing unstable bits (since search engines can drop them in the repo first). 
+
+The below table shows the pertinent information for installs on the repo landing page.  
+
+| Property             	| Description                                      	|
+|----------------------	|--------------------------------------------------	|
+| Installation targets 	| Native installers; scripts for curl installs   	|
+| Source branch        	| rel/1.0.0                                        	|
+| Linked builds        	| Latest green build of rel/1.0.0                  	|
+
+# Acquisition modes
+There are multiple acquisition modes that the CLI will have:
+
+1. Native installers
```

Comment:
@piotrpMSFT unless you start considering the dependencies. Then manual way is really, **really** manual.